apis/core.oam.dev/v1alpha2/core_scope_types.go

```diff
@@ -47,8 +47,24 @@ type HealthScopeSpec struct {
 	// ProbeInterval is the amount of time in seconds between probing tries.
 	ProbeInterval *int32 `json:"probe-interval,omitempty"`
 
+	// AppRefs records references of applications' components
+	AppRefs AppReferences `json:"appReferences,omitempty"`
```

Comment:
```suggestion
	AppRefs ComponentsReferences `json:"appReferences,omitempty"`
```
Seems you are referencing to components.